src/components-examples/angular/search/search-autocomplete/search-autocomplete-example.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+/**
+ * @title Search Autocomplete
+ * @order 20
+ */
+@Component({
+  selector: 'sbb-search-autocomplete-example',
+  templateUrl: './search-autocomplete-example.html',
+})
+export class SearchAutocompleteExample {
+  searchControl = new FormControl('');
+  searchValues: string[] = [];
+
+  filteredOptions = options.slice(0);
+
+  handleSearch(value: string) {
+    this.searchValues = [value, ...this.searchValues].slice(0, 5);
```

Comment:
I think slicing is an overkill for the showcase examples and can only confuse consumers?

Reply:
I don't want the list to grow unlimited. Should I maybe add a comment to explain why this is done?

Reply:
Also okay for me